raiden/network/proxies/payment_channel.py

```diff
@@ -27,33 +24,12 @@ class PaymentChannel:
     def __init__(
         self,
         token_network: TokenNetwork,
-        channel_identifier: ChannelID,
+        channel_state: NettingChannelState,
```

Comment:
Note: This adds a dependency from the state machine into the proxies. Ideally it would use a different type, but it was faster to just reuse the one from the node's state. if the need ever arises this can be changes.